python/pyarrow/tests/test_parquet.py

```diff
@@ -29,21 +29,19 @@
 from pyarrow.compat import guid, u, BytesIO, unichar
 from pyarrow.tests import util
 from pyarrow.filesystem import LocalFileSystem
-import pyarrow as pa
 from .pandas_examples import dataframe_with_arrays, dataframe_with_lists
 
 import numpy as np
 import pandas as pd
-
+import pyarrow as pa
+import pyarrow.parquet as pq
 import pandas.util.testing as tm
 
 # Ignore these with pytest ... -m 'not parquet'
-parquet = pytest.mark.parquet
+pytestmark = pytest.mark.parquet
```

Comment:
A nit, but "pytestmark" is informative. Call it "parquet_test" perhaps?

Reply:
This is a module level marker, see the end of [this section](https://docs.pytest.org/en/latest/example/markers.html#marking-whole-classes-or-modules)

Reply:
Gasp. Can you add a comment?

Reply:
Sure